database/migrations/2021_04_25_230328_create_event_distance_for_members.php

```diff
@@ -17,8 +17,8 @@ public function up()
             $table->id();
             $table->unsignedBigInteger('event_id')->default(0);
             $table->unsignedBigInteger('user_id')->default(0);
-            $table->unsignedDecimal('miles',8,2)->default(0);
-            $table->unsignedDecimal('minutes',8,2)->default(0);
+            $table->decimal('miles',8,2)->default(0);
```

Comment:
Unsigned decimal is no longer supported, and in fact was never finalized in MySQL.

Reply:
Fortunately this feature was never finished. So it'll be shcrödinger's feature.